browser-test/src/admin_predicates.test.ts

```diff
@@ -221,6 +221,9 @@ describe('create and edit predicates', () => {
     await adminQuestions.addTextQuestion({
       questionName: 'depends on previous',
     })
+    await adminQuestions.addTextQuestion({
```

Comment:
I think you meant for this to be a DateQuestion, I'd move it up with the existing one too.

Reply:
Also update the name to be more readable "on or after"